cmd/login.go

```diff
@@ -62,24 +72,27 @@ type LoginCmd struct {
 }
 
 func NewLoginCmd(cmd *cobra.Command, args []string) (*LoginCmd, error) {
-	server, err := cmd.Flags().GetString(argRootServer)
-	if err != nil {
-		return nil, err
+	viper.BindPFlag(argRootServer, cmd.Flags().Lookup(argRootServer))
+	viper.BindPFlag(argLoginUsername, cmd.Flags().Lookup(argLoginUsername))
+	viper.BindPFlag(argLoginPassword, cmd.Flags().Lookup(argLoginPassword))
+	server := viper.GetString(argRootServer)
```

Comment:
The documentation for viper is a bit confusing, so I'm just making sure I got it right: This *falls back* to the config file and command line arguments take precedence, right?

Reply:
Yes, exactly :)

I also had to do a manual test, to figure out which was the default here